modules/custom/express_dashboards/express_dashboard_user/templates/express-dashboard-user-profile.tpl.php

```diff
@@ -1,4 +1,10 @@
 <div class="express-dashboard-user-profile-wrapper">
+  <div class="col-md-2 pull-right">
+    <?php
+      $block = module_invoke('user', 'block_view', 'online');
```

Comment:
I generally think it's a bad idea to call functions other than formatting/rendering in a template file. You could add the block content to express_dashboard_user_content() so you wouldn't have to load the block in the template.

Also, I think more styling could be added to this page the SEO dashboard has more styling as an example.